File: app/src/main/java/be/kdg/mobile_client/user/model/Notification.java
package be.kdg.mobile_client.user.model;

import java.time.LocalDateTime;

import lombok.Data;
import lombok.Setter;

@Data
public class Notification {
    private int id;
    private String message;
    private NotificationType type;
    private LocalDateTime timestamp;
    @Setter
    private boolean read;
    private String ref;
}

File: web-client/src/app/model/notification.ts
import {NotificationType} from './notificationType';
import {User} from './user';
import {Time} from '@angular/common';

export class Notification {
  id: number;
  message: string;
  type: NotificationType;
  read: boolean;
  timestamp: string;
  ref: string;
}
